### DISCUSSION TITLE
What about this tool requires running with local admin?

### DISCUSSION THREAD

**Discussion by meckerlejsquare** [rink-turksma/IntunePrepTool#5] [Q&A] [answered]
Just curious what function of the tool requires local admin.  I like the tool but giving anything system access always makes me nervous.

**Comment by rink-turksma** [comment, marked as answer]
While making a package. Testing the install and uninstall scripts is one of the steps.

**Comment by rink-turksma** [comment]
> Just curious what function of the tool requires local admin. I like the tool but giving anything system access always makes me nervous.

The installation of the required PowerShell modules currently requires authentication because the "-Scope AllUsers" option is used. Additionally, the installation of Winget itself requires elevation. I will consider changing this in the future and testing install and uninstall script can maybe be done with prompting for elevation when the "Test install" button is clicked.
I need to do some testing for this. Thanks for your feedback!
